src/frontends/onnx/tests/tests_python/test_backend.py

```diff
@@ -683,6 +685,22 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
     ),
 ]
 
+if platform.system() == 'Darwin':
+    tests_expected_to_fail.extend([
+        (
+            skip_issue_58676,
```

Comment:
why is it skip_issue_58676 and not xfail_issue_58676 ?

Reply:
Seems like it is inconsistent on macOS as we have seen on master with #20367 and #20402: one pipeline was okay, but the next was failing in the post-commit because of the test accidentally passing and failing the pipeline with `xpass`.